src/neo-vm/Types/InteropInterface.cs

```diff
@@ -15,16 +15,16 @@ public InteropInterface(object value)
             _object = value ?? throw new ArgumentException();
         }
 
-        public override bool Equals(object obj)
+        public override bool Equals(StackItem other)
         {
-            if (ReferenceEquals(this, obj)) return true;
-            if (obj is InteropInterface i) return _object.Equals(i._object);
+            if (ReferenceEquals(this, other)) return true;
+            if (other is InteropInterface i) return _object.Equals(i._object);
```

Comment:
`ReferenceEquals` for _object?

Reply:
If `_object` is a reference type, then it automatically uses `ReferenceEquals()`.